WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletViewModel.cs

```diff
@@ -117,7 +117,7 @@ public ConnectHardwareWalletViewModel(WalletManagerViewModel owner) : base("Hard
 					Logger.LogInfo("Creating a new wallet file.");
 					var walletName = Global.WalletManager.WalletDirectories.GetNextWalletName("Coldcard");
 					var walletFullPath = Global.WalletManager.WalletDirectories.GetWalletFilePaths(walletName).walletFilePath;
-					KeyManager.CreateNewHardwareWalletWatchOnly(mfp, extPubKey, walletFullPath);
+					Global.WalletManager.AddWallet(KeyManager.CreateNewHardwareWalletWatchOnly(mfp, extPubKey, walletFullPath));
```

Comment:
nit: please add a WalletManager property instead of using global..

even if we set it from global initially.

Reply:
What is the point?

Reply:
We are trying to remove Global from all the viewmodels and pass in just what they need...

WalletManager, UiConfig. Global isnt needed most of the time.

Reply:
We have done this on most other VMs we have modified, and we should do it here.

Reply:
ACK

Reply:
Sooner or later WalletManager will be decoupled from Global so better like this.